core/metaproperty.h

```diff
@@ -15,6 +15,7 @@
 #define GAMMARAY_METAPROPERTY_H
 
 #include "gammaray_core_export.h"
+#include <common/metatypedeclarations.h>
```

Comment:
related or?

Reply:
also could not build without this but no longer seems to be an issue in the current commit. can revert

Reply:
reverted